libqrexec/qrexec.h

```diff
@@ -136,7 +136,7 @@ struct trigger_service_params {
 struct trigger_service_params3 {
     char target_domain[64];           /* null terminated ASCII string */
     struct service_params request_id; /* service request id */
-    // char service_name[0];          /* null terminated ASCII string, size = msg_header.len - sizeof(struct trigger_service_params3) */
+    char service_name[];          /* null terminated ASCII string, size = msg_header.len - sizeof(struct trigger_service_params3) */
```

Comment:
This was commented out on purpose: https://github.com/QubesOS/qubes-core-qrexec/pull/5#discussion_r282036625